probably/pgcl/check.py

```diff
@@ -374,14 +504,15 @@ def check_expression(program, expr: Expr) -> Optional[CheckFail]:
 
     .. doctest::
 
-        >>> program = Program(list(), dict(), dict(), list())
-        >>> check_expression(program, FloatLitExpr("1.0"))
+        >>> from .ast import ProgramConfig
+        >>> program = Program(ProgramConfig(), list(), dict(), dict(), dict(), list())
```

Comment:
This incomprehensible list of constructor arguments shows that there's something wrong with the API of the `Program` class.